pkg/provider/cloud/openstack/provider.go

```diff
@@ -289,22 +289,26 @@ func reconcileNetwork(ctx context.Context, netClient *gophercloud.ServiceClient,
 		}
 	}
 
-	// If NetworkName not specified, Create network with name kubernetes-clusterid
+	// If NetworkName not specified, Create network with name kubernetes-clusterid.
 	if networkName == "" {
```

Comment:
This could be the else-path to the if above.

Reply:
oh my bad! done!